src/app/pages/ecommerce/index.js

```diff
@@ -11,7 +11,7 @@ import {
 
 import "@newfold-labs/wp-module-ecommerce";
 import "@newfold-labs/wp-module-ecommerce/crazydomain.css";
-import "@newfold-labs/wp-module-ecommerce/styles.scss";
```

Comment:
Why are we commenting this out? Is it no longer needed?

Reply:
Yea, they changed how they load them in, it's a file that's no longer in the module. I was just testing locally to fix the build and forgot to remove it. I'll remove it rather than comment it out.